apps/src/Sounds.js

```diff
@@ -111,7 +124,7 @@ Sounds.prototype.initializeAudioUnlockState_ = function () {
  */
 Sounds.prototype.isAudioUnlocked = function () {
   // Audio unlock doesn't make sense for the fallback player as used here.
-  return this.audioUnlocked_ || !this.audioContext;
+  return this.audioUnlocked_;
```

Comment:
Oh!  This can also happen if `window.AudioContext` and `window.webkitAudioContext` are both falsy (so, we're using the fallback player).  Maybe this can check for `window.AudioContext` here?

Alternatively, just a flag for 'have we even tried to unlock the audio yet' probably captures the original intent of this check.